test.sh

```diff
@@ -20,14 +28,15 @@ if [ "x$CURRENT_ARCH" == "x" ]; then
     exit 1
 fi
 
+killall secretshare-server
```

Comment:
This makes me a bit uneasy.  There's a reason I store the server PID below and then specifically kill it later.